tests/app/views/test_audits.py

```diff
@@ -116,6 +116,38 @@ def test_should_get_audit_event_by_object(self):
         assert_equal(len(data['auditEvents']), 1)
         assert_equal(data['auditEvents'][0]['user'], 'rob')
 
+    def test_should_get_audit_event_by_missing_object(self):
+        self.add_audit_events_with_db_object()
+
+        response = self.client.get('/audit-events?object-type=suppliers&object-id=100000')
+        data = json.loads(response.get_data())
+
+        assert_equal(response.status_code, 200)
+        assert_equal(len(data['auditEvents']), 0)
+
+    def test_should_only_get_audit_event_with_correct_object_type(self):
+        self.add_audit_events_with_db_object()
+
+        with self.app.app_context():
+            supplier = Supplier.query.filter(Supplier.supplier_id == 1).first()
+            event = AuditEvent(
+                audit_type=AuditTypes.supplier_update,
+                db_object=supplier,
+                user=1,
```

Comment:
It isn't obvious from the code how this test works. I think the idea is that add_audit_events adds one Supplier-event, then we add another event with the sane object ID but with Service as the object type, and then check that only the original Supplier-event is returned? I'm not sure that checking the user is 'rob' is a good test for this - is user=1 definitely not 'rob' too? If so it would be nice to have it clearer in the code that the second audit event does not have 'rob' as the user.